app/models/pipeline_run.rb

```diff
@@ -645,7 +645,7 @@ def load_taxons(downloaded_json_path, refined = false)
     update_is_phage
 
     # rm the json
-    _stdout, _stderr, _status = Open3.capture3("rm -f #{downloaded_json_path}")
+    _stdout, _stderr, _status = Open3.capture3("rm", "-f", downloaded_json_path)
```

Comment:
Did your tests include checking whether this file actually got removed?

Reply:
yep 👍